aea/cli/create.py

```diff
@@ -180,7 +179,7 @@ def _create_agent_config(ctx: Context, agent_name: str, set_author: str) -> Agen
         license_=DEFAULT_LICENSE,
         registry_path=os.path.join("..", DEFAULT_REGISTRY_PATH),
         description="",
-        default_ledger=DEFAULT_LEDGER,
+        default_ledger=None,
```

Comment:
as the AEA has not the key generation anymore, I think it makes more sense to have it nullable (but mandatory).

Reply:
In theory, this can be reverted while keeping the rest.

Reply:
I don't agree. An AEA needs a ledger, so we might as well set the default one, as we do already. Then `aea install` installs the relevant plugin.

Reply:
that doesn't solve the issue of no key present in the AEA project, even if all the plugins are installed. Reason is that there isn't anymore the key generation.

Reply:
Yes, a simple error message informing the user of the missing key is needed.